src/components/Menu/Menu.component.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from 'react';
+
+import { PageType } from '../../App.tsx';
+import darkMoon from '../../assets/darkMoon.png';
+import darkSun from '../../assets/darkSun.png';
+import lightMoon from '../../assets/lightMoon.png';
+import lightSun from '../../assets/lightSun.png';
+import separatorSVG from '../../assets/separator.svg';
+import { ThemeContext } from '../ThemeProvider/ThemeProvider.component.tsx';
+
+import styles from './Menu.module.css';
+
+interface NameComponentProps {
+    setActivePage: (page: PageType) => void;
+}
+
+export const MenuComponent: React.FC<NameComponentProps> = ({ setActivePage }) => {
+    const { setLightTheme, setDarkTheme, theme } = useContext(ThemeContext);
+    const handleButton = (name: PageType) => {
+        setActivePage(name);
+        localStorage.setItem('activePage', name);
+    };
+    const activePage = localStorage.getItem('activePage');
+
+    const lightModeIcon = () => (theme === 'light' ? lightSun : darkSun);
```

Comment:
You need just 2 svg for sun and moon. Change color with css [Fills_and_Strokes](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Fills_and_Strokes)